hetmech/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for hetmech project.
+
+Generated by 'django-admin startproject' using Django 2.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '#o0ew9x2)es%z8(7+ev(g^508ere9g-a9d^(ckv86t3rb--sx4'
```

Comment:
This key should be confidential. Usually it is set saved somewhere else (in a file that will be ignored by git), then `setting.py` will read that file and save the key. 

The file that is ignored by git will also include other confidential information, such as database password, or even database user name and port number, etc.

Here is an example of how Tribe web server sets them up:
https://github.com/greenelab/tribe/blob/6844a3525ac694f9ee8f52c2f8396297afbbc92b/tribe/settings.py#L59